### PR TITLE
Replaced ConfigServiceProvider with Dotenv

### PR DESCRIPTION
**Pull request by wjzijderveld**
To allow for an easier deployment process, we want to use environment variables to configure the API.
With the current config.json file, this is not ideal.

By using [vlucas/phpdotenv], we open up the possibility to set or override configuration with environment variables.

[vlucas/phpdotenv]: https://packagist.org/packages/vlucas/phpdotenv

### CODE REVIEW

File: app/bootstrap.php [hunk 1]
```diff
@@ -23,12 +23,13 @@
 
 $app = new Application();
 
-$configFile = __DIR__ . '/../config.json';
-if (! file_exists($configFile)) {
-    $configFile = __DIR__ . '/../config.json.dist';
-}
+$env = new \Dotenv\Dotenv(__DIR__ . '/../');
+$env->load();
```
Comment (asm89) [inline app/bootstrap.php]: Let's add some validation here? https://github.com/vlucas/phpdotenv#requiring-variables-to-be-set

File: app/bootstrap.php [hunk 2]
```diff
@@ -23,12 +23,13 @@
...
 
-$app->register(new ConfigServiceProvider($configFile));
+$app['debug']                 = getenv('TOGGLE__DEBUG');
+$app['allowed_origins']       = json_decode(getenv('TOGGLE__ALLOWED_ORIGINS'));
```
Comment (asm89) [inline app/bootstrap.php]: Can you throw if the decode failed? Users will have a bad time because of this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Changelog for Qandidate toggle API
+
+## 0.3.x
+
+- Replaced configuration with `vlucas/phpdotenv`. With this method there is [more flexibility to deploy] the API
```
Comment (asm89) [inline CHANGELOG.md]: Replaced `igorw/...` with `vlucas/phpdotenv`

**Comment by asm89** [comment]
Also, can you nuke `config.json.dist` + squash your commits?

**Comment by asm89** [comment]
:+1:

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Changelog for Qandidate toggle API
+
+## 0.3.x
+
+- Replaced `igorw/config-service-provider` with `vlucas/phpdotenv`. With this method there is [more flexibility to deploy] the API
+  `.env.dist` gives you the same configuration as `config.json.dist` gave you before.
```
Comment (fritsjanb) [inline CHANGELOG.md]: The config.json's `redis_uri` is replaced with `TOGGLE__REDIS_DSN`, aka `uri` with `dsn`. Might be worth mentioning!

Comment (wjzijderveld) [inline CHANGELOG.md]: Added a note about it!

**Comment by othillo** [comment]
:+1:

**Pull request merged**